src/main/java/org/sitenv/ccdaparsing/model/CCDARefModel.java

```diff
@@ -154,6 +155,12 @@ public CCDAFamilyHistory getFamilyHistory() {
 		return familyHistory;
 	}
 
+	public void setMedicationEquipments(CCDAMedicalEquipment medicalEquipment) {
```

Comment:
do you mind if we reweite this as:

```java
public void setMedicalEquipment(CCDAMedicalEquipment medicalEquipment) {**
	this.medicalEquipment = medicalEquipment;
}
```

Equipments is technically not a word. Nor does its Medication prefix match the variable name, medicalEquipment. Can we also rename the getter in the same manner to:
```java
public CCDAMedicalEquipment getMedicalEquipment() {
```
If this does not make sense for some reason (using MedicalEquipment). Can we then rename the class itself, to CCDAMedicationEquipment vs CCDAMedicalEquipment and then remove the plural form on the getter and the setter.
Let me know if any questions or disagreements. Thanks

Reply:
we have made those changes